deployment/ansible/roles/raster-foundry.worker/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+worker_home: /opt/worker
+worker_deploy_branch: "master"
+
+worker_config:
```

Comment:
This appears to duplicate the configuration settings inside of the `app` role. I think we should try moving both sets into the `base` role to avoid duplication.